recycleIT-backend/src/main/java/com/recycleit/recycleitbackend/entity/User.java

```diff
@@ -0,0 +1,22 @@
+package com.recycleit.recycleitbackend.entity;
+
+
+import javax.persistence.Id;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+
+@Entity
```

Comment:
@Table would be required, you can leave as is and add it when start implementing repository layer